src/main/java/core/basesyntax/service/FruitOperationServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.FruitOperationDao;
+import core.basesyntax.model.FruitOperation;
+
+public class FruitOperationServiceImpl implements FruitOperationService {
+    private FruitOperationDao fruitOperationDao;
+
+    public FruitOperationServiceImpl(FruitOperationDao fruitOperationDao) {
+        this.fruitOperationDao = fruitOperationDao;
+    }
+
+    @Override
+    public void createNewFruitOperation(String fruit) {
+        FruitOperation fruitOperation = new FruitOperation();
+        fruitOperation.setFruit(fruit);
+        fruitOperationDao.add(fruitOperation);
+    }
+}
```

Comment:
The `FruitOperationServiceImpl` class seems to be tightly coupled with the `FruitOperationDao`. According to the SOLID principles, particularly the Dependency Inversion Principle, it would be better to depend on an abstraction rather than a concrete implementation. Consider using an interface for the DAO layer if it's not already the case.